<?xml version="1.0"?>
<launch>

  <arg name="verbose" default="false"/>

  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="$(find nps_uw_multibeam_sonar)/worlds/sonar_tank_oculus_m1200d_nps_multibeam.world"/>
    <arg name="verbose" value="$(arg verbose)"/>
  </include>

  <!-- rqt_image_view for image_raw_sonar -->
  <node name="rqt_image_view_sonar" pkg="rqt_image_view"
        type="rqt_image_view" args="/oculus_m1200d/sonar_image">
  </node>



</launch>
